package.json

```diff
@@ -58,8 +58,10 @@
     "releaseNotesUrl": "https://grafana.com/docs/grafana/next/release-notes/"
   },
   "lint-staged": {
+    "*.{js,ts,tsx,betterer}": [
```

Comment:
this fake betterer extension makes me a little concerned tbh 😅 what happens if it's removed?

Reply:
if you remove it then you will write an invalid JSON (two keys the same)

Reply:
Looking at the globs in betterer.ts could we remove the `js` and `betterer` extensions here instead?

```
export default {
  'better eslint': () =>
    countEslintErrors()
      .include('**/*.{ts,tsx}')
      .exclude(/public\/app\/angular/),
  'no undocumented stories': () => countUndocumentedStories().include('**/!(*.internal).story.tsx'),
};
```

Reply:
This makes much more sense. I'll do that better.